core/web/health_controller.go

```diff
@@ -96,13 +99,202 @@ func (hc *HealthController) Health(c *gin.Context) {
 			Output: output,
 		})
 	}
+	slices.SortFunc(checks, func(a, b presenters.Check) int {
+		return cmp.Compare(a.Name, b.Name)
+	})
+
+	switch c.NegotiateFormat(gin.MIMEJSON, gin.MIMEHTML, gin.MIMEPlain) {
+	case gin.MIMEJSON:
+		break // default
+
+	case gin.MIMEHTML:
+		if err := newCheckTree(checks).WriteHTMLTo(c.Writer); err != nil {
+			hc.App.GetLogger().Errorw("Failed to write HTML health report", "err", err)
+			c.AbortWithStatus(http.StatusInternalServerError)
+		}
+		return
 
-	if testing.Testing() {
+	case gin.MIMEPlain:
 		slices.SortFunc(checks, func(a, b presenters.Check) int {
```

Comment:
Should this sort be removed since you sort before the switch statement now?

Reply:
yeah